src/vs/base/common/filters.ts

```diff
@@ -378,10 +378,10 @@ export function matchesFuzzy2(pattern: string, word: string): IMatch[] | null {
 	return score ? createMatches(score) : null;
 }
 
-export function anyScore(pattern: string, lowPattern: string, patternPos: number, word: string, lowWord: string, wordPos: number): FuzzyScore {
+export function anyScore(pattern: string, lowPattern: string, patternPos: number, word: string, lowWord: string, wordPos: number, options: FuzzyScoreOptions = FuzzyScoreOptions.default): FuzzyScore {
 	const max = Math.min(13, pattern.length);
 	for (; patternPos < max; patternPos++) {
-		const result = fuzzyScore(pattern, lowPattern, patternPos, word, lowWord, wordPos, { firstMatchCanBeWeak: false, boostFullMatch: true });
+		const result = fuzzyScore(pattern, lowPattern, patternPos, word, lowWord, wordPos, options);
```

Comment:
This is likely not even needed. We use `anyScore` only to "guess" highlights for labels when suggestions use a filter-text that's different than the label-text. In those cases we do the filtering on one string but need to do the highlight on another. There is no correctness needed, it's best-effort and could have relaxed rules already before

Reply:
Good to know! Removed updates here.